src/app/grants/page.tsx

```diff
@@ -1,4 +1,5 @@
 import { CardLayout } from '@/components/CardLayout'
+import { CardStepProcess } from '@/components/CardStepProcess'
```

Comment:
I know it's a mouthful but until we refactor the Cards, let's go with `GrantsApplicationProcessCard`